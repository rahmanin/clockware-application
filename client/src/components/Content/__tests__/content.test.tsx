import {Content} from "../index"
import React from 'react';
import {cleanup, render} from '@testing-library/react';

afterEach(cleanup);

test('Render', () => {
  const component = render(
    <Content children/>,
  );
  expect(component.baseElement).toMatchSnapshot();
  
});