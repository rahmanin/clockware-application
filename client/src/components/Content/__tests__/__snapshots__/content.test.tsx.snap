// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render 1`] = `
<body>
  <div>
    <div
      class="content"
    />
  </div>
</body>
`;
